index.js

```diff
@@ -148,7 +149,7 @@ Router.matchPath = function(url, route) {
 
 Router.onPopstate = function(e) {
   // routerStarted = false;
-  var loc = window.location.pathname + window.location.search;
```

Comment:
Yes because Router.go adds it. It makes url "tagging?tag=favorite" then router.go appends another "?tag=favorite" after it. 

Therefore it is safe to take out